admin/tests/unit/routes/authenticated/target-profiles/list_test.js

```diff
@@ -113,4 +114,73 @@ module('Unit | Route | authenticated/target-profiles/list', function (hooks) {
       });
     });
   });
+
+  module('#beforeModel', function (hooks) {
+    let router;
+
+    hooks.beforeEach(function () {
+      router = this.owner.lookup('service:router');
+      router.transitionTo = sinon.stub();
+    });
+
+    test('it should allow an admin member with "SUPER_ADMIN" role to access the page', function (assert) {
```

Comment:
Je me pose des questions sur cette façon de tester car on est en train de tester finalement le comportement de `this.accessControl.restrictAccessTo` et sans le stubber.
Il faudrait plutôt faire comme ça `admin/tests/unit/routes/authenticated/organizations/new_test.js` et vérifier que dans le beforeModel on appelle bien `restrictAccessTo` avec les bons paramètres